tests/test_enterprise/views/test_enterprise_proxy_login_view.py

```diff
@@ -133,13 +142,17 @@ def test_tpa_redirects_using_tpa_hint_param(
             identity_provider.enterprise_customer = enterprise_customer
             identity_provider.save()
 
-        url = self._get_url_with_params(enterprise_slug_override=enterprise_customer.slug, tpa_hint=tpa_hint_param)
+        url = self._get_url_with_params(
+            enterprise_slug_override=enterprise_customer.slug,
+            tpa_hint=tpa_hint_param,
+        )
         response = self.client.get(url)
         query_params = QueryDict(mutable=True)
         next_url = self.next_url
 
         if redirected_tpa_hint:
```

Comment:
Again, see my previous comment above- it's not clear to me if we need to have tpa_hint at 2 possible levels of redirect (where before we only had 1) or if your replacement for a single level of redirect is fixing something that was built incorrectly the first time. I'd just like a double check?